controllers/ingress/group_controller.go

```diff
@@ -324,6 +323,18 @@ func (r *groupReconciler) setupWatches(_ context.Context, c controller.Controlle
 	return nil
 }
 
+func (r *groupReconciler) getActiveAndInactiveIngressObjects(ingGroup ingress.Group) ([]metav1.Object, []metav1.Object) {
```

Comment:
seems have a utility like below is more clean :D 
```
func ToSliceOfMetaObject[T any](s []T) []metav1.Object {
    result := make([]metav1.Object, len(s))
    for i, v := range s {
        result[i] = v
    }
    return result
}
```
then we can simply use ```ToSliceOfMetaObject(ingGroup.Members)``` and ```ToSliceOfMetaObject(ingGroup. InactiveMembers)```

Reply:
generic needs type constraints, the assignment result[i] = v doesn't work with any. I've updated the PR with utils.